instance/static/html/instance/index.html

```diff
@@ -49,9 +50,11 @@
                 title="{{ server.status_description }}">
             {{ server.status_name }}
```

Comment:
I don't think we need to display the server status and the instance status separately here. And I like the way the server status appears now more than the icon used for the instance status.

So maybe we could combine these?

```
      <h3>
        {{ selected.instance.name }}
        <span class="status radius label"
          ng-class="{warning: !selected.instance.status.is_healthy, success: selected.instance.status.is_healthy}"
          tooltip="{{ selected.instance.status_description }}">
            <i ng-attr-class="{{ selected.instance.is_steady && (selected.instance.is_healthy && 'fa fa-check' || 'fa fa-times') || 'fa fa-ellipsis-h' }}"></i>
            {{selected.instance.status_name}}
        </span>
      </h3>
```

And then below in the `UL.instance-description`, we could add a new `<li>` that lists each server and states its IP and status.

```
              <li class="instance-server-list">
                Servers: 
                <span ng-repeat="server in selected.instance.active_server_set">
                  vm{{server.id}} ({{server.status_name}})
                </span>
              </li>
```

End result:
![screen shot 2016-04-24 at 3 08 59 pm](https://cloud.githubusercontent.com/assets/945577/14770779/989817ba-0a2e-11e6-825e-8618fe01701a.png)

Which is much nicer than:
![screen shot 2016-04-24 at 3 10 26 pm](https://cloud.githubusercontent.com/assets/945577/14770785/b45e9e42-0a2e-11e6-9313-a4f4ae628a9a.png)